__tests__/app.test.js

```diff
@@ -38,4 +38,37 @@ describe("GET /api/topics", () => {
       });
     });
   });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("return 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("return an article object", () => {
+    return request(app).get("/api/articles/2").then(({body}) => {
+      expect(body.article).toContainAllKeys(["article_id", "title", "topic", "author", "body", "created_at", "votes", "article_img_url"]);
```

Comment:
Here I recommend the use of .toMatchObject or .objectContaining so the value types can be checked as well at the same time with 'expect.any'.